datacontract/export/dbt_converter.py

```diff
@@ -135,14 +135,13 @@ def _supports_constraints(model_type):
 def _to_columns(fields: Dict[str, Field], supports_constraints: bool, adapter_type: Optional[str]) -> list:
     columns = []
     for field_name, field in fields.items():
-        column = _to_column(field, supports_constraints, adapter_type)
-        column["name"] = field_name
+        column = _to_column(field_name, field, supports_constraints, adapter_type)
         columns.append(column)
     return columns
 
 
-def _to_column(field: Field, supports_constraints: bool, adapter_type: Optional[str]) -> dict:
-    column = {}
+def _to_column(field_name: str, field: Field, supports_constraints: bool, adapter_type: Optional[str]) -> dict:
+    column = {"name": field_name}
```

Comment:
We do not need to fix the test case in this pull request, because python's dict does not consider the order when comparing.

- https://github.com/datacontract/datacontract-cli/blob/main/tests/test_export_dbt_sources.py#L134